src/builder.ts

```diff
@@ -1438,13 +1484,13 @@ export function makeSchema(options: SchemaConfig): GraphQLSchema {
   if (shouldGenerateArtifacts) {
     // Generating in the next tick allows us to use the schema
     // in the optional thunk for the typegen config
-    new TypegenMetadata(options).generateArtifacts(schema).catch((e) => {
-      console.error(e);
-    });
+    new TypegenMetadata(builder, options)
+      .generateArtifacts(schema)
+      .catch((e) => {
+        console.error(e);
+      });
```

Comment:
🚲 🏠 
```
.catch(console.error)
```

Reply:
`error` isn't bound to console, so it'd need to be `.catch(console.error.bind(console))`, I'd prefer the arrow.

Reply:
I think you're thinking of the browser, not node:

```
> Promise.reject(new Error('oops')).catch(console.error)
Promise { <pending> }
> Error: oops
    at repl:1:16
    at Script.runInThisContext (vm.js:126:20)
    at REPLServer.defaultEval (repl.js:384:29)
    at bound (domain.js:420:14)
    at REPLServer.runBound [as eval] (domain.js:433:12)
    at REPLServer.onLine (repl.js:700:10)
    at REPLServer.emit (events.js:214:15)
    at REPLServer.EventEmitter.emit (domain.js:476:20)
    at REPLServer.Interface._onLine (readline.js:316:10)
    at REPLServer.Interface._line (readline.js:693:8)
```

Reply:
Huh, apparently the browsers (Chrome) changed since I last did this stuff (been a while):

![image](https://user-images.githubusercontent.com/284476/66223956-beb2f380-e6a2-11e9-9549-befd6c7af444.png)

Firefox too... well all the better then.